flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/FileChannelManagerImplTest.java

```diff
@@ -211,7 +211,7 @@ public static void main(String[] args) throws Exception {
             if (callerHasHook) {
                 // Verifies the case that both FileChannelManager and its upper component
                 // have registered shutdown hooks, like in IOManager.
-                ShutdownHookUtil.addShutdownHook(() -> manager.close(), "Caller", LOG);
+                ShutdownHookUtil.addShutdownHook(manager, "Caller", LOG);
```

Comment:
I don't think this test drives the changed code.

Reply:
I don't understand what you said. Could you tell me more?

Reply:
sorry. The changed code is in   method ```
public IOManagerAsync(String[] tempDirs, ExecutorService executorService) {
        super(tempDirs, executorService);
```

So to test this we need to drive this method. I think you have a test that tests the same pattern but not the actual changed code.